client/components/steps/business-address.jsx

```diff
@@ -54,6 +58,18 @@ module.exports = React.createClass( {
 		SetupProgressActions.submitBusinessAddress( this.state );
 	},
 
+	renderInstallWooCheckbox: function() {
+		if ( ! this.state.sell_online ) {
+			return null;
+		}
+		return (
+			<div className="welcome__business-address--woo-container">
+				<input className="welcome__business-address--checkbox" type="checkbox" name="install_woo" id="install-woo" checked={ this.state.install_woo } onChange={ this.handleChange } />
```

Comment:
`id="install-woo"` should probably be changed to `id="install_woo"` , as it currently does not match the `for` attribute of the corresponding label.